source/lex.h

```diff
@@ -451,18 +429,78 @@ auto expand_string_literal(
     );
 
     //  Put the final non-interpolated chunk straight into ret
-    if (
-        first
-        || current_start < std::ssize(text)-1
-        )
-    {
-        add_plus(true);
-        ret += text.substr(current_start);
+    if (current_start < std::ssize(text)-1) {
+        parts.add_string(text.substr(current_start, std::ssize(text)-current_start-1));
     }
 
-    return ret;
+    return parts.generate();
 }
 
+auto expand_raw_string_literal(
+    const std::string& opening_seq,
+    const std::string& closing_seq,
+    string_parts::adds_sequences closing_strategy,
+    std::string_view text,
+    std::vector<error>& errors,
+    source_position src_pos) -> string_parts
+{
+    auto const length = std::ssize(text);
+    auto pos = 0;
+    auto first_quote_pos = pos;
```

Comment:
Ditto, unused?

Reply:
Ah, that is an artifact from the previous implementation - I have corrected my compiler flags to spot unused variables.

Do you have a list of flags that you have enabled during compilation? I will make sure I have the same set of flags set not to send faulty changes.

I have prepared the fix and I am running local tests to ensure that I did not break anything.

Reply:
Fixed